VocaDbWeb/Views/Search/Partials/_TagSearchList.cshtml

```diff
@@ -34,7 +34,7 @@
 			<tr>
 				<td style="width: 80px">
 					<a data-bind="visible: $data.mainPicture && mainPicture.urlSmallThumb, attr: { href: vdb.utils.EntryUrlMapper.details_tag(id, urlSlug), title: $data.additionalNames }" href="#">
-						<img data-bind="attr: { src: ($data.mainPicture ? mainPicture.urlSmallThumb : '') }" title="Cover picture" class="coverPicThumb img-rounded" />
+						<img data-bind="attr: { src: ($data.mainPicture ? mainPicture.urlSmallThumb : '') }" title="Cover picture" class="coverPicThumb img-rounded" referrerpolicy="same-origin" />

```

Comment:
But surely these images are never off-site?

Reply:
Yeps, I was doing quite blindly when adding the referrerpolicy tag as I was not 100% definite about what each picture actually is. Supposingly only song thumbs need this; but with the `same-origin` attribution, images already from our own domain wouldn't be effected, so over-tagging would only be a tidiness issue. I've removed the two unnecessary occurances you mentioned.